NOTICE

```diff
@@ -0,0 +1,18 @@
+NOTICE
+
+This repository contains both source code and data, each under separate licenses:
+
+1. **Source Code** 
+   Licensed under the **Apache License, Version 2.0**.
+   See the [LICENSE](./LICENSE) file for the full license text.
+
+2. **CI Dataset** (`plugins/nf-co2footprint/src/resources/ci_data`)
+   Licensed under the **Open Data Commons Open Database License (ODbL) v1.0**.
+   See [CI LICENSE](plugins/nf-co2footprint/src/resources/ci_data/LICENSE) for the full license text.
+
+Redistributions of this repository must comply with the terms of both licenses where applicable.
+
+If you use or redistribute the data, please credit the source and comply with the ODbL attribution requirements.
+If you use or redistribute the code, follow the Apache 2.0 licensing and attribution requirements.
```

Comment:
It almost seems as if there is something missing for the CI data, since the attribution case is explicitly mentioned. What would you think about:
 
```suggestion
If you use, modify or redistribute the CI data, you must credit the original source and comply with the attribution, share-alike, and open-access requirements of the ODbL.
If you use, modify or redistribute the code, follow the Apache 2.0 licensing and attribution requirements.
```